special-pages/pages/new-tab/app/favorites/components/Favorites.module.css

```diff
@@ -1,18 +1,22 @@
 .root {
     width: 100%;
-    margin: 0 auto var(--sp-4);
+    margin: 0 auto var(--ntp-gap);
```

Comment:
I've inverted the logic here. Now the the spacing is consistent to start with, then only if we are showing the expander will we increase/decrease margins (below)